src/rdb_protocol/ql2.proto

```diff
@@ -412,7 +412,7 @@ message Term {
         // Map a function over a sequence and then concatenate the results together.
         CONCAT_MAP = 40; // Sequence, Function(1) -> Sequence
         // Order a sequence based on one or more attributes.
-        ORDER_BY   = 41; // Sequence, (!STRING | Ordering)... -> Sequence
+        ORDER_BY   = 41; // Sequence, (!STRING | Ordering)..., {index: (!STRING | Ordering)} -> Sequence
```

Comment:
In implementing a json client, I'm not sure whether this would say `[41, [sequence, {index: "id"}]]` or `[41, [sequence], {index: "id"}]` (the latter is the correct form).